irods/session.py

```diff
@@ -158,6 +159,13 @@ def port(self):
 
     @property
     def server_version(self):
+        try:
+            reported_vsn = os.environ.get("PYTHON_IRODSCLIENT_REPORTED_SERVER_VERSION")
+            return tuple(ast.literal_eval(reported_vsn))
+        except:
```

Comment:
Consider making `except:` into `except Exception:` (or perhaps `KeyError`?) or whatever exception is being handled here to appease Codacy

Reply:
Done!